lib/DDG/Spice/GoWatchIt.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::GoWatchIt;
+
+use DDG::Spice;
+
+name "GoWatchIt Search";
+source "GoWatchIt.com";
+description "Find out where to watch your favorite movies and shows!";
+category "entertainment";
+topics "entertainment";
+code_url "https://github.com/plexusent/zeroclickinfo-spice/blob/gwi/lib/DDG/Spice/GoWatchIt.pm";
+attribution github  => ['https://github.com/plexusent', 'GoWatchIt.com'],
+            web     => ['http://gowatchit.com'],
+            twitter => ['gowatchit', 'GoWatchIt.com'];
+
+my @triggers = ('watch', 'stream', 'watch online', 'on demand', 'watch now', 'stream online', 'buy movie', 'rent movie');
+my @killwords = ('movie', 'show', 'tv', 'online', 'stream');
+
+my $killwords = join '|', @killwords;
+
+triggers startend => @triggers;
+
+spice to => 'http://gowatchit.com/api/v3/search?term=$1&full_meta=true&api_key={{ENV{DDG_SPICE_GOWATCHIT_APIKEY}}}';
+spice wrap_jsonp_callback => 1;
+
+handle remainder => sub {
+  return if !$_;
+
+  $_ = lc $_; # Make sure everything is lower case.
+
+  $_ =~ s/$killwords//g; # remove killwords
```

Comment:
What about queries like "watch scary movie"? Wouldn't "movie" get removed? Another one is "watch showrunners"--it would only search for "runners".